_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -55,15 +55,17 @@ To set up an alert for the detector at this stage of detector creation, continue
 
 1. In the **Trigger name** box, enter a name for the trigger.
 1. To define rule matches for the alert, select security rules, severity levels, and tags.
-<img src="{{site.url}}{{site.baseurl}}/images/Security/alert_rules.png" alt="Rules used to define an alert">
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/alert_rules.png" alt="Rules used to define an alert">
```

Comment:
Global comment: instead of using `<br>`, just add a new line. Sometimes the use of `<br>` is unavoidable, but adding a new line is the preferred method.